spark/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala

```diff
@@ -526,32 +526,6 @@ class StatsCollectionSuite
         )
       }
     }
-
-    test(s"Delta statistic column: invalid data type $invalidType in nested column") {
```

Comment:
remove this test, no longer applicable since we shouldn't throw an error https://github.com/delta-io/delta/pull/4531/files#diff-2af271647c445a4476d202c2b2a08fd22e8ea41ad1da85153812f750c52d52abL468